packages/lib/getBrandColours.tsx

```diff
@@ -1,7 +1,9 @@
 import { useBrandColors } from "@calcom/embed-core/embed-iframe";
 
-const BRAND_COLOR = "#292929";
-const DARK_BRAND_COLOR = "#fafafa";
+import { DEFAULT_DARK_BRAND_COLOR, DEFAULT_LIGHT_BRAND_COLOR } from "./constants";
+
+const BRAND_COLOR = DEFAULT_LIGHT_BRAND_COLOR;
+const DARK_BRAND_COLOR = DEFAULT_DARK_BRAND_COLOR;
```

Comment:
Use the values from constants -> These colours get loaded here https://github.com/calcom/cal.com/pull/13167/files#diff-abbbafbce3959191cbdfce95ed71dc38466b75d1695dd78dc81a66e4117e1d76L171-L172 if there is no brand colour passed in